hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/OzoneDelegationTokenSecretManager.java

```diff
@@ -221,8 +216,6 @@ public Token<OzoneTokenIdentifier> createToken(Text owner, Text renewer,
 
   /**
    * Add delegation token in to in-memory map of tokens.
-   * @param token
-   * @param ozoneTokenIdentifier
```

Comment:
LGTM +1
Just one small question, Is there any specific reason we made changes to the JavaDocs?

Reply:
> Is there any specific reason we made changes to the JavaDocs?

This was done as part of separate cleanup commit (4759560).

`@param`/`@throws` without description is invalid and provides no additional information to the method signature.

Reply:
Got it! thanks for clarifying